.github/workflows/windows-BUILD_DEPS.yml

```diff
@@ -12,19 +12,24 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest]
-        use_ortools_stable : ["OFF"]
 
     steps:
-    - uses: actions/checkout@v2      
+    - uses: actions/checkout@v2
+    
+    - name: Download pre-compiled librairies
+      run: |
+           choco install wget
+           wget https://github.com/AntaresSimulatorTeam/antares-deps/releases/download/v1.0.0/rte-antares-deps-${{matrix.os}}-x64-Release-all.zip
+           Expand-Archive ./rte-antares-deps-${{matrix.os}}-x64-Release-all.zip -DestinationPath .
+           rm rte-antares-deps-${{matrix.os}}-x64-Release-all.zip
+           
+    - name: Init submodule
+      run: |
+           git submodule update --init --recursive src
 
     - name: Configure
       run: |
-           cmake -B _build -S src -DBUILD_DEPS=ON -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON -DUSE_ORTOOLS_STABLE=${{ matrix.use_ortools_stable }}
-    
-    - name: Remove dependency compilation directory
-      uses: JesseTG/rm@v1.0.2
-      with:
-        path: _build/build_deps
+           cmake -B _build -S src -DDEPS_INSTALL_DIR=rte-antares-deps-Release -DCMAKE_BUILD_TYPE=release -DBUILD_TESTING=ON -DBUILD_OPENSSL=ON -DBUILD_curl=ON -DBUILD_wxWidgets=ON
```

Comment:
Need hack with -DBUILD_OPENSSL=ON -DBUILD_curl=ON -DBUILD_wxWidgets=ON to be able to used precompiled dependencies.